src/main/java/org/apache/commons/io/input/CloseShieldReader.java

```diff
@@ -50,4 +54,16 @@ public void close() {
         in = ClosedReader.CLOSED_READER;
     }
 
+    /**
+     * Creates a proxy that shields the given reader from being
+     * closed.
+     *
+     * @param in underlying reader
+     * @return the created proxy
+     * @since 2.9.0
+     */
+    public static CloseShieldReader wrap(final Reader in) {
```

Comment:
as see other Javadoc comments, in this case `in` -> `reader`

Reply:
Changed as suggested